src/CMakeLists.txt

```diff
@@ -102,6 +114,9 @@ endif (MOAB_FOUND)
 set_source_files_properties(decay.cpp PROPERTIES COMPILE_FLAGS 
     "-O0")
 
+add_executable(alphad ${PROJECT_SOURCE_DIR}/src/ensdf_processing/ALPHAD/alphad.f)
+target_link_libraries(alphad pyne)
+
```

Comment:
Is ndsflib actually being built?  It seems that it is only built if spatial
solvers is also built.

Be Well
Anthony

On Tue, Apr 21, 2015 at 7:15 PM Josh[ua] Howland <notifications@github.com>
wrote:

> In src/CMakeLists.txt
> <https://github.com/pyne/pyne/pull/714#discussion_r28839852>:
>
> > @@ -102,6 +114,9 @@ endif (MOAB_FOUND)
> >  set_source_files_properties(decay.cpp PROPERTIES COMPILE_FLAGS
> >      "-O0")
> >
> > +add_executable(alphad ${PROJECT_SOURCE_DIR}/src/ensdf_processing/ALPHAD/alphad.f)
> > +target_link_libraries(alphad pyne)
> > +
>
> When I attempt to use the executable, the exact error is:
> josh@josh-ThinkPad-W530:~/Desktop/ENDSF_FOR_TESTING/ALPHAD_LINUX$ ./alphad
> ./alphad: symbol lookup error: ./alphad: undefined symbol: date_20_
> date_20_ is a reference to the SUBROUTINE DATE_20(Date) in nsdflib95.f. I
> poked around google/stack overflow quite a bit, but I have not been able to
> figure out why the target_link_libraries(alphad pyne) line doesn't link
> the alphad executable and nsdflib95.o.
>
> When I use gfortran to build the executable, it works as desired, so its
> definitely something I'm doing wrong in cmake. Sorry for the delay in
> responding - I wanted to try to figure this out myself before coming back
> here again, but I haven't gotten anywhere so I'm back.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyne/pyne/pull/714/files#r28839852>.
>